pkgs/applications/networking/qv2ray/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv
+, fetchFromGitHub
+, qmake
+, qttools
+, wrapQtAppsHook
+, cmake
+, clang
+, grpc
+, protobuf
+, openssl
+, pkgconfig
+, c-ares
+, abseil-cpp
+, libGL
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qv2ray";
+  version = "2.5.0";
+
+  src = fetchFromGitHub {
+    owner = "Qv2ray";
+    repo = "Qv2ray";
+    rev = "v${version}";
+    sha256 = "12x72rsd5iaj06qmma8x6g7syqaiixnfb37f3ppx8x00c77s78va";
+    fetchSubmodules = true;
+  };
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DQV2RAY_DISABLE_AUTO_UPDATE=on"
+    "-DQV2RAY_TRANSLATION_PATH=${placeholder "out"}/share/qv2ray/lang"
+  ];
+
+  preConfigure = ''
+    export _QV2RAY_BUILD_INFO_="Qv2ray Nixpkgs"
+    export _QV2RAY_BUILD_EXTRA_INFO_="(Nixpkgs build) nixpkgs"
+  '';
+
+  enableParallelBuilding = true;
```

Comment:
This is not required, nix figures this out automatically. As the nixpkgs manual states:
`Unless set to false, some build systems with good support for parallel building including cmake, meson, and qmake will set it to true.`